docs/controls/Expander.md

```diff
@@ -53,13 +52,6 @@ The `ExpandDirection` property can take 4 values that will expand the content ba
 * `Right` - from left to right
 * `Left` - from right to left
 
-### DisplayMode
-
-The `DisplayMode` property can take 2 values that will modify the behavior of the Expander control:
-
-* `Expand` - the content is either visible or collapsed (default)
-* `Overlay` - when the content is collapsed, an overlay content is shown
```

Comment:
I don't think the ContentOverlay container property is actually called out elsewhere?  So, we probably need a little blurb on it still somewhere, eh?

Reply:
Good point, added